tests/ctl/conftest.py

```diff
@@ -67,321 +38,26 @@ def monkeypatch_requests(test_client, monkeysession) -> None:
-                    name="declaration-name",
-                    data_categories=[],
-                    data_use="provide",
-                    data_subjects=[],
-                    data_qualifier="aggregated_data",
-                    dataset_references=[],
-                )
-            ],
-            system_dependencies=[],
-        ),
-    }
-    yield resources_dict
-
-
-@pytest.fixture()
-def test_manifests() -> Generator:
-    test_manifests = {
-        "manifest_1": {
-            "dataset": [
-                {
-                    "name": "Test Dataset 1",
-                    "organization_fides_key": 1,
-                    "datasetType": {},
-                    "datasetLocation": "somedb:3306",
-                    "description": "Test Dataset 1",
-                    "fides_key": "some_dataset",
-                    "datasetTables": [],
-                }
-            ],
-            "system": [
-                {
-                    "name": "Test System 1",
-                    "organization_fides_key": 1,
-                    "systemType": "mysql",
-                    "description": "Test System 1",
-                    "fides_key": "some_system",
-                }
-            ],
-        },
-        "manifest_2": {
-            "dataset": [
-                {
-                    "name": "Test Dataset 2",
-                    "description": "Test Dataset 2",
-                    "organization_fides_key": 1,
-                    "datasetType": {},
-                    "datasetLocation": "somedb:3306",
-                    "fides_key": "another_dataset",
-                    "datasetTables": [],
-                }
-            ],
-            "system": [
-                {
-                    "name": "Test System 2",
-                    "organization_fides_key": 1,
-                    "systemType": "mysql",
-                    "description": "Test System 2",
-                    "fides_key": "another_system",
-                }
-            ],
-        },
-    }
-    yield test_manifests
-
-
-@pytest.fixture()
-def populated_manifest_dir(test_manifests: Dict, tmp_path: str) -> str:
-    manifest_dir = f"{tmp_path}/populated_manifest"
-    os.mkdir(manifest_dir)
-    for manifest in test_manifests.keys():
-        with open(f"{manifest_dir}/{manifest}.yml", "w") as manifest_file:
-            yaml.dump(test_manifests[manifest], manifest_file)
-    return manifest_dir
-
-
-@pytest.fixture()
-def populated_nested_manifest_dir(test_manifests: Dict, tmp_path: str) -> str:
-    manifest_dir = f"{tmp_path}/populated_nested_manifest"
-    os.mkdir(manifest_dir)
-    for manifest in test_manifests.keys():
-        nested_manifest_dir = f"{manifest_dir}/{manifest}"
-        os.mkdir(nested_manifest_dir)
-        with open(f"{nested_manifest_dir}/{manifest}.yml", "w") as manifest_file:
-            yaml.dump(test_manifests[manifest], manifest_file)
-    return manifest_dir
-
-
 @pytest.fixture
-def db() -> Generator:
```

Comment:
not a big deal, but just curious - why the removal of the return type annotation here? does it cause problems?